docs/apache-airflow/troubleshooting.rst

```diff
@@ -20,6 +20,19 @@
 Troubleshooting
 ===============
 
+Obscure scheduling failures
```

Comment:
I rather we won't place this here.

Our goal should be shorting this doc not adding more stuff to it.

I rather we explain the mechanisem of scheduler-executor dynamic in the relevant section if the docs and then explain the limits/edge cases.

Reply:
This particular log message (`"could not queue task %s (still running after %d attempts)"`) is essentially useless. The troubleshooting page adds context that we can't fit into a log message. It'd be difficult to provide this context in a way users can find in the normal documentation.

Reply:
But you need to consider that people land in the doc without hitting the message you are refering to.

I wonder if maybe the message should refer to github discussion arround the problem. I am not a fan of troubleshooting pages. If this is a bug/limitation/problem we don't have a good solution for then it's an open issue.